index.js

```diff
@@ -97,6 +97,15 @@ const buildParserFlags = ({flags, booleanDefault}) => {
 			delete flag.isMultiple;
 		}
 
+		if (Array.isArray(flag.aliases)) {
+			if (flag.alias) {
+				flag.aliases.push(flag.alias);
+			}
+
+			flag.alias = flag.aliases;
+			delete flag.aliases;
+		}
```

Comment:
This will also allow an empty array:

```js
flags: {
	unicorn: {
		type: 'string',
		aliases: []
	}
}
```

Should this be an error?

Reply:
> Should this be an error?

No